dice.cpp

```diff
@@ -0,0 +1,33 @@
+#include "dice.h"
+
+Dice::Dice(){
+    sum =0;
+}
```

Comment:
Technically, there's no problem. But it is best practice to initialize all your fields in the constructor.